documentation/howto/import_FH_app.adoc

```diff
@@ -0,0 +1,189 @@
+
+.index.html File Edited
+image::index_html_file_edited.png[index.html File Edited]
+
+[[test_app]]
+== 3. Test the Application
+
+To test the imported application, right-click the application and then click `Run As` > `Run w/remote FeedHenry server`.
+
+.FeedHenry Application Edited
+image::FeedHenry_app_edited.png[FeedHenry Application Edited]
+
+The changes made to the `index.html` file are reflected on the simulated device. Click a corner of the displayed device to rotate it in that direction. Alternatively, right-click the simulated device and click `Rotate Right` or `Rotate Left` as to rotate it in the desired direction. 
+To view the application on a different CordovaSim simulated device, right-click the device and click `Skin`. From the list of skins, select a skin to view the application. 
+
+[[push_changes_to_FH]]
+== 4.  Push the Changes Back to the FeedHenry Server
+
+Use the following instructions to push changes to the application back to the FeedHenry server: 
+
+. In the `Project Explorer` view, right-click the application name.
+. Click `Team` > `Commit`.
+. In the `Commit Changes` window, `Commit message` field, type a message for the commit. 
+. In the `Files` field, select the files that you have edited and want to push to the server and then click `Commit and Push`.
+. In the `Push Results _[application name]_` window, ensure all the details are correct and click `OK`.
+. Log into FeedHenry at `https://_[your-studio-domain]_.feedhenry.com`.
+. Click `Projects`.
+. Click the `Project Title` under which your application is located and then click the application.
+
+*Result:* The simulated device in the `App Preview` section displays the change that you have just pushed to the FeedHenry server.
+
+.FeedHenry Application Edited and Displayed on the FeedHenry Server
+image::FeedHenry_app_edited_on_FeedHenry_server.png[FeedHenry Application Edited and Displayed on the FeedHenry Server]
+
+== Did You Know 
+
+* Add a new API key to your FeedHenry account by clicking `Add New Key` and then following the on-screen instructions.
+
+* Set up your SSH key in the FeedHenry account by clicking `Settings` > `SSH Key Management` and then following the on-screen instructions.
+
+== Troubleshooting 
+
+=== Git Communication Error 
+
+.Git Communication Error
+image::FeedHenry_git_communication_error.png[Git Communication Error]
+
+*Error Message*
+
+Problem when cloning the application. This can be due to a network problem or missing security credentials. Refer to error log for details.
+
+**Issue**
+
+When a FeedHenry account is set up, the user’s API Keys are configured by default, but the SSH Public key must be manually configured. Importing a FeedHenry application into JBoss Developer Studio means that the application repository is accessed via Git. Without a Public SSH key, the IDE is unable to complete the action and this error appears.
```

Comment:
"the tools are unable to complete the action" or "it is not possible to complete the action" ?